packages/merge-on-green/src/merge-on-green.ts

```diff
@@ -331,6 +432,77 @@ handler.checkPRMergeability = async function checkPRMergeability(
   }
 };
 
+//TODO: change the search query if expanding the allowlist to search for more authors
+/**
+ * For a given repository, looks through all the PRs and checks to see if they have a MOG label
+ * @param owner the owner of the repo
+ * @param repo the repo name
+ * @param context the context of the webhook payload
+ * @returns void
+ */
+handler.pickUpPR = async function pickUpPR(
+  owner: string,
+  repo: string,
+  github: Context['github']
+) {
+  // Github does not support searching the labels with 'OR'.
+  // The searching for issues is considered to be an "AND" instead of an "OR" .
+  const issuesAutomergeLabel = await github.paginate(
```

Comment:
These 2 searches can probably be started at the same time then awaited before we want to start executing on them.